### PR TITLE
LPD-18581- Disable Object Validation when the script management setting is false

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPD-18581

### CODE REVIEW

**Comment by danvb112** [comment]
Hey @igor-franca, this PR is a dependency of https://github.com/liferay-objects/liferay-portal/pull/3129, so you only need to review the last three commits :smile:

**Review by igor-franca — APPROVED** [review]
No related errors could be found 😄

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/ModalAddObjectValidation.tsx [hunk 1]
```diff
@@ -36,13 +37,23 @@ const initialValues: Partial<ObjectValidation> = {
...
 	objectValidationRuleEngines,
+	scriptManagementEnabled,
 	setShowAddObjectRelationshipModal,
 }: ModalAddObjectValidationProps) {
 	const [error, setError] = useState<string>('');
 	const {observer, onClose} = useModal({
 		onClose: () => setShowAddObjectRelationshipModal(false),
 	});
 
+	let newObjectValidationRuleEngines = objectValidationRuleEngines;
```
Comment (rebsilva) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/ModalAddObjectValidation.tsx:48]: same comment from [here](https://github.com/liferay-objects/liferay-portal/pull/3129#discussion_r1520211272)

but if you think this won't cause a problem or a bug, you can leave the way it is!

**Review by rebsilva — COMMENTED** [review]
hey @danvb112! just a comment, let me know if it makes sense or not

**Comment by rebsilva** [comment]
i still need to test this PR, so please don't forward it; thanks!

**Review by rebsilva — APPROVED** [review]
LGTM! :tada: amazing work @danvb112! :cherry_blossom:

**Comment by rebsilva** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
Sent to: https://github.com/brianchandotcom/liferay-portal/pull/148187

**Pull request closed without merging**